### ISSUE TITLE
Plenary Sessions (under Schedule)

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#23]
**Number of volunteers**: 1

- [x] Edit the `sitedata_acl2020/speakers.csv`.
   * Get the keynote schedule [here](https://acl2020.org/program/keynotes/).
   * Ask @dreasysnail or @raosudha89 for more information.
- [x] Change "Example Day" to "Keynotes", and remove "Highlighted Papers" section in `templates/schedule.html`.

**Comment by raosudha89** [comment]
Suggestion from PCs
- [ ] Make this a plenary session instead and include Keynotes, Test of Time Award, Lifetime achievement award and other livestream

**Comment by georgepar** [comment]
> Suggestion from PCs
> 
> * [ ]  Make this a plenary session instead and include Keynotes, Test of Time Award, Lifetime achievement award and other livestream

Not sure how to do this. Any hints?

**Comment by hao-fang** [comment]
@georgepar Let's do this. 

Create a sub-page for each day during July 6 -- 8, like ICLR2020 did (https://iclr.cc/virtual_2020/calendar.html#tab-Monday). See the screenshot below.
![image](https://user-images.githubusercontent.com/9957482/84966954-aca9a180-b0c7-11ea-8c88-2ee7baa82870.png)

Each sub-page contains the plenary sessions on that day.  Here is our schedule.

## July 6
![image](https://user-images.githubusercontent.com/9957482/84967089-fbefd200-b0c7-11ea-951b-55d167bd28d3.png)

## July 7 
![image](https://user-images.githubusercontent.com/9957482/84967096-014d1c80-b0c8-11ea-9a16-842795d76d48.png)

## July 8
![image](https://user-images.githubusercontent.com/9957482/84967123-1033cf00-b0c8-11ea-8f24-290c737e63a5.png)


Let us know if you have any thoughts about this.

**Comment by georgepar** [comment]
@hao-fang 
Hi Hao,

Can you share the plenary schedule so I can include them and close this? 
Basically we need something that is close to the fields of [this YAML](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/master/speakers.yml)

**Comment by hao-fang** [comment]
@georgepar See https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/33 and https://github.com/acl-org/acl-2020-virtual-conference/pull/145

The rough schedule is here
![image](https://user-images.githubusercontent.com/9957482/85210770-9b46dc00-b2f7-11ea-9688-1662a3462dd7.png)


We will need to improve it.
- [ ] Rename the "Speakers" to "Plenary Sessions" in the page title
- [ ] Order the sessions by time (I didn't do the PST / GMT conversion)
- [ ] Gather the necessary information to be displayed. Can you make a list? I'll send an email to our program chairs and cc you to get those information.

Also, I'm thinking instead of opening a new page for each plenary session, can we make those as a collapse cards? (https://getbootstrap.com/docs/4.3/components/collapse/) In this way, the audience will stay on the same page for the whole plenary sessions on this day. (Just like a physical room experience).
But let's first get the basic version working and come back to this when we have time, unless you already know an easy way to do it.
Feel free to create another ticket if you feel you need some additional volunteers to work on this together.

**Comment by hao-fang** [comment]
@OguzCennet Are you actively working on this ticket? I feel there may be a way to split the ticket into separate sub-tasks.

**Comment by raosudha89** [comment]
In the schedule tab, under each of the days (Monday, Tuesday, Wednesday), I think it is more useful to show the events in the form of a calendar so that people can understand what time is the event in their own timezone. Just listing it out might be less useful.

We can have an "Overall Calendar" that shows the events for all days. And then additionally show each day separately again in form of a calendar. 

Since we have many events in parallel, it is getting a little crowded in the overall calendar. So having a calendar for each day may make it more cleaner.

**Comment by hao-fang** [comment]
@raosudha89 We may test whether the calendar view can show enough context information about the event. The title itself may not be informative enough. The current "card" view have more opportunity to show the start-end time, speaker, and some description about the meeting.
But if you or some volunteer can implement a calendar view, then we can see how it works. Because of the current time constraint, we can probably only get a basic version working.

**Comment by raosudha89** [comment]
@hao-fang  Sounds good. I will wait for the workshop multiple calendar view issue https://github.com/acl-org/acl-2020-virtual-conference/pull/153 to be fixed and then I will create a demo for this and share it here.

**Comment by raosudha89** [comment]
All plenary sessions have been added to the schedule (https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/49)
Closing this issue and creating a new one to add content of plenary session https://github.com/acl-org/acl-2020-virtual-conference/issues/205
